routes/game.rb

```diff
@@ -11,7 +11,11 @@ class Api
 
         # '/api/game/<game_id>/'
         r.is do
-          game.to_h(include_actions: true)
+          game_data = game.to_h(include_actions: true)
```

Comment:
instead could we make this an option on the to_h, so to_h takes in a param who it's for